decidim-core/app/views/decidim/account/_password_fields.html.erb

```diff
@@ -1,2 +1,2 @@
-<%= form.password_field :password, value: form.object.password, autocomplete: "off", help_text: t("devise.passwords.edit.password_help", minimun_characters: ::PasswordValidator::MINIMUM_LENGTH) %>
+<%= form.password_field :password, value: form.object.password, autocomplete: "new-password", help_text: t("devise.passwords.edit.password_help", minimun_characters: ::PasswordValidator::MINIMUM_LENGTH) %>
```

Comment:
The `autocomplete` option does not do anything when using the `password_field` helper:
https://github.com/sgruhier/foundation_rails_helper/blob/master/lib/foundation_rails_helper/form_builder.rb#L53

The foundation helper will force it to `off`.

We need to resolve this. One way is that you can override that method in the Decidim form builder:
https://github.com/decidim/decidim/blob/d750a81a0aad944313030f461baaa6638f1d8adf/decidim-core/lib/decidim/form_builder.rb

As per chromium istructions, we should also apply the `new-password` value to the password confirmation input as well to make it compatible with the password management functionality of browsers and extensions:
https://www.chromium.org/developers/design-documents/form-styles-that-chromium-understands/

In the HTML standard discussion, they also provide the same advice:
https://github.com/whatwg/html/issues/3531

![The intent is just to use new-password twice.](https://user-images.githubusercontent.com/864340/166501463-1f5e4de1-a15e-4372-a962-98d20b0302f5.png)